strr-base-web/app/composables/useStrrApi.ts

```diff
@@ -15,10 +17,17 @@ export const useStrrApi = () => {
       })
     }
     // TODO: add type filter in call (need in api first)
-    const resp = await $strrApi<{ registrations: T[] }>('/registrations')
-    return type
-      ? resp.registrations?.filter(reg => reg.registrationType === type)
-      : resp.registrations
+    const resp = await $strrApi<{ registrations: T[], total: number }>('/registrations', {
+      query: {
+        limit,
+        offset
+      }
+    })
+    if (type) {
+      const filtered = resp.registrations?.filter(reg => reg.registrationType === type)
```

Comment:
Filtering in UI for now?

Reply:
Yes for now, the API endpoint does not currently accept `registrationType` query filter.
But this should be done as a follow up task.